### PR TITLE
LPS-151251 Remove URIBuilder and use HttpComponentsUtil for usages

### PR DESCRIPTION
**Pull request by julschong**
Hi Tina,

Main Task: [LPS-147928](https://issues.liferay.com/browse/LPS-147928)
SubTask: [LPS-151251](https://issues.liferay.com/browse/LPS-151251)

Thank you!

### CODE REVIEW

**Comment by julschong** [comment]
@catarineaguiar Hi Catarine,
could you please help me scan the test result?
Thank you!

**Comment by catarineaguiar** [comment]
@julschong no regressions found.

**Comment by catarineaguiar** [comment]
Hi @julschong, there is a semver failure:

```
[exec] com.liferay.semantic.versioning.SemanticVersioningTest > testSemanticVersioning[/apps/petra/petra-apache-http-components] FAILED
     [exec]     java.lang.AssertionError: [Baseline Warning] Bundle Version Change Recommended: 2.0.0
```

**Comment by julschong** [comment]
forced pushed to update bundle version to 2.0.0

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
@catarineaguiar Hi Catarine,
I think the semver issue is resolved. Thank you!
Could you please help me double check to make ok to go?
Thank you

**Comment by catarineaguiar** [comment]
@julschong it is OK now!

**Comment by julschong** [comment]
Hi Tina, @tinatian
please help review the PR when you can.
Thank you!

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina.

**Comment by tinatian** [comment]
@julschong Code conflicts, please rebase.

Thanks.
Tina.

**Comment by julschong** [comment]
pushed to resolve conflict

File: modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/ExportTranslationDisplayContext.java [hunk 1]
```diff
@@ -282,29 +280,27 @@ private JSONObject _getExportFileFormatJSONObject(
...
-					"classPK", String.valueOf(_getDraftLayoutPlid(classPK)));
+				url = HttpComponentsUtil.addParameter(
+					url, "classPK",
+					String.valueOf(_getDraftLayoutPlid(classPK)));
 			}
 			else {
-				uriBuilderWrapper.addParameter(
-					"classPK", String.valueOf(classPK));
+				url = HttpComponentsUtil.addParameter(
+					url, "classPK", String.valueOf(classPK));
```
Comment (julschong) [inline modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/ExportTranslationDisplayContext.java:298]: conflict here

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina.

**Comment by shuyangzhou** [comment]
ci:forward

**Pull request closed without merging**